ideascube/configuration/registry.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+
+
+# Please try and keep this stored in alphabetical order
+REGISTRY = {
+    # 'namespace': {
+    #     'key': {
+    #         'python_type': str,
+    #         'default': 'Default value',
+    #     },
+    # },
+    'server': {
+        'site-name': {
+            'python_type': str,
```

Comment:
Next pull request adds a « pretty type » which I have called `type`.

I wanted to be explicit that this one is the actual Python type.

Plus it's an implementation detail, not something anybody would ever type outside of the `Configuration.get` and `Configuration.set` methods.